projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -450,8 +448,6 @@ else if (!settings.printParseTree()) {
 
    private BatfishLogger _logger;
 
-   private SortedMap<String, String> _questionMap;
```

Comment:
This was used for debugging. I believe I also had a future purpose in mind. But for now I really don't care if you remove it.